locobot/robot/slam_pkg/README.md

```diff
@@ -13,14 +13,17 @@ roscore &
 
 ```
```

Comment:
can you also add details about the slam implementation in the readme here? Add references to papers /  implementations where necessary. How does it work? What are its gaps? What can be the future improvements? etc.